test/OpenTelemetry.Exporter.Geneva.Tests/GenevaMetricExporterTests.cs

```diff
@@ -765,17 +780,7 @@ private void CheckSerializationForSingleMetricPoint(Metric metric, GenevaMetricE
             {
                 if (bucket.BucketCount > 0)
                 {
-                    if (bucket.ExplicitBound != double.PositiveInfinity)
```

Comment:
nit: I think we need not create a new method for this as this is much simpler than the actual exporter code. IMO, it also makes it a little simpler when debugging failed test cases to not jump around from function to function.